api/organisations/templates/organisations/api_flags_blocked_notification.txt

```diff
@@ -0,0 +1,7 @@
+Hi there,
+
+As per previous warnings, we have blocked organisation {{ organisation.name }}. Please visit app.flagsmith.com to upgrade your account and re-enable service.
+
+Thank you!
+
+The Flagsmith Team
```

Comment:
Note that this template will likely need updating as per [this conversation](https://docs.google.com/document/d/1nqNj5kdqCbcyc29v0FmMORt0eZCY5hIxgzwxJLttPi8/edit?disco=AAABRskXWjg).

Reply:
```suggestion
Hi there,

This is a system generated notification related to your Flagsmith API Usage. As per previous warnings, we have had to blocked your companyorganisation {{ organisation.name }} after the 7 day grace period. Flags are not currently being served for your organization, and will continue to be blocked until your billing period resets or you upgrade your account. You can upgrade your account at app.flagsmith.com.

Thank you!

The Flagsmith Team
```

(Note that this will also need to be changed in the HTML template as well)

Reply:
Ok this has been added